### PR TITLE
Update access.md

### PR DESCRIPTION
**Pull request by Harrycheng233**
Resolves #[Add issue number here]

 Changes:

 Screenshots of the change:

#### PR Checklist

- [ ] `npm run lint` passes
- [ ] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

File: contributor_docs/zh/access.md [hunk 1]
```diff
@@ -1,40 +1,58 @@
-# 我们关注的重点是可及性
-
-在[2019年贡献者大会](https://p5js.org/community/contributors-conference-2019.html)上，我们承诺只向p5.js添加增加可及性（包括包容性和/或无障碍性）的功能。这意味着要考虑多样性的方向（如性别、社会、经济、种族、族裔、语言、残疾等），这些方向可能影响到可及性/参与度，并采取行动来承认、消除和预防障碍。我们将优先考虑历史上受边缘化群体的需求，而不是为了使相对特权的群体在使用p5.js时更加舒适。
-
-我们不会接受不支持增加可及性努力的功能请求。您将在我们的问题和拉取请求模板中看到这些标准的体现。
+<!-- 我们对可及性的承诺对贡献者和库的用户意味着什么。 -->
 
-这是关于p5.js中访问性和包容性的持续对话的一部分。我们的意图是将这些作为p5.js构建的核心价值观，这一点在我们的[社区声明](../CODE_OF_CONDUCT.md)中有所说明，该声明是在[2015年贡献者会议](https://p5js.org/community/contributors-conference-2015.html)上编写的。
+# 我们关注的重点是可及性
```
Comment (sproutleaf) [inline contributor_docs/zh/access.md:3]: 我觉得Access应该翻译成可访问性或网页无障碍？@unicar9 你觉得呢？

Comment (sproutleaf) [reply contributor_docs/zh/access.md:3]: 哦哦哦等下 我看到下文写的access (inclusion and accessibility)了 看来无障碍性accessibility和access还是得区分开 还是等unicar的意见哦！

File: contributor_docs/zh/access.md [hunk 2]
```diff
@@ -1,40 +1,58 @@
...
 
-**请将此视为一个起点。** 我们希望邀请更多关于可及性意味着什么以及如何优先考虑可及性的讨论。
+在[ 2019 年贡献者大会](https://p5js.org/community/contributors-conference-2019.html)上，我们承诺只向 p5.js 添加提高可及性（包括包容性和无障碍性）的功能。我们不会接受那些不支持这些努力的功能请求。我们致力于认知、消除和预防障碍的工作。这意味着要考虑在交叉[^1]体验中可能影响可及性和参与度的多样性方向。这包括了性别、种族、族裔、性别倾向、语言、地理位置等因素的考虑。我们会优先考虑边缘群体的需求，而不是仅维护在 p5.js 社区中特权群体的长期舒适体验。我们正共同探索和研究可及性的含义，并积极学习该如何实践和传授可及性相关的知识。我们选择通过广泛、交叉和联合的框架来思考可及性。这一承诺是我们在[社区声明](https://p5js.org/community/)中概述的 p5.js 核心价值观的一部分。
```
Comment (sproutleaf) [inline contributor_docs/zh/access.md:5]: 认知、消除->承认、废除

File: contributor_docs/zh/access.md [hunk 3]
```diff
@@ -1,40 +1,58 @@
...
 
-以下是我们认为可以增加可及性的努力的示例：
+- 将文档和其他资料翻译成更多的语言，从而解构语言帝国主义[^4]（例如，Rolando Vargas 的[用库纳语编程](https://medium.com/@ProcessingOrg/culture-as-translation-processing-in-kuna-language-with-rolando-vargas-and-edinson-izquierdo-8079f14851f7)，Felipe Santos Gomes, Julia Brasil, Katherine Finn Zander, 和 Marcela Mancino 的[为葡萄牙语用户的 Pê Cinco：国际化与普及](https://medium.com/processing-foundation/translating-p5-js-into-portuguese-for-the-brazilian-community-14b969e77ab1)）
+- 改进对辅助技术的支持，比如屏幕阅读器（例如，Katie Liu 的[在 p5.js 中添加 Alt 文本](https://medium.com/processing-foundation/adding-alt-text-e2c7684e44f8)，Claire Kearney-Volpe 的[ p5 可及性项目](https://medium.com/processing-foundation/p5-accessibility-115d84535fa8)）
+- 遵循我们工具中的[网络内容可访问性指南](https://www.w3.org/TR/WCAG21/)，并致力于让用户在其项目中更轻松地遵守这些准则
+- 让 p5.js 的错误信息对使用该工具的人士更为友好和获取更多支持。（例如，[ p5.js 友好错误系统(FES)](https://github.com/processing/p5.js/blob/main/contributor_docs/friendly_error_system.md)）
+- 在创意编程和数字艺术领域历史上被排除和边缘化的社区内指导和支持 p5.js 的学习者
+- 举办社区活动（例如，[ p5.js 访问日2022](https://p5js.org/community/p5js-access-day-2022.html)，[我们想要的网络：p5.js x W3C TPAC 2020](https://medium.com/processing-foundation/p5-js-x-w3c-tpac-bee4c621a053)），采用以可及为中心的组织策略（例如，美国手语翻译，实时字幕，无障碍场地）
```
Comment (sproutleaf) [inline contributor_docs/zh/access.md:38]: 感觉可以把p5.js访问日中的访问换成access的翻译，和上文同步！

File: contributor_docs/zh/access.md [hunk 4]
```diff
@@ -1,40 +1,58 @@
...
 ## 维护
+我们不接受那些不支持我们增加可及性努力的功能请求。你将在我们的 issue 和 PR 模板中看到这一标准的体现。我们还确认了保持 p5.js 现有功能集的意图。我们愿意修复代码库中任何区域的错误。我们相信工具的一致性将使其对初学者更加易用。提升易用性的功能请求示例包括：
+为使用性能较低硬件的人提升性能（例如，支持向帧缓冲区绘制/从帧缓冲区读取）
+API 的一致性（例如，添加 arcVertex() 函数以通过 beginShape()/endShape() 创建弧线）
+
+___
+
+请将此视为一份“不断发展中的文档”。我们将持续讨论并优先考虑可及性的含义。我们邀请我们的社区参与讨论这份文档及其所描述的价值观。如果你有任何想法或建议，欢迎通过在 Github 上提交 issue 或通过发送电子邮件至 hello@p5js.org 与我们分享。
+
+这个版本的 p5.js 可及性声明是在2023年开源艺术贡献者大会上，与 Evelyn Masso、Nat Decker、Bobby Joe Smith III、Sammie Veeler、Sonia (Suhyun) Choi、Xin Xin、Kate Hollenbach、Lauren Lee McCarthy、Caroline Sinders、Qianqian Ye、Tristan Jovani Magno Espinoza、Tanvi Sharma、Tsige Tafesse 和 Sarah Ciston 的合作下修订的。它在 Bobby Joe Smith III 和 Nat Decker 的处理基金会奖学金支持下完成并发布。
```
Comment (sproutleaf) [inline contributor_docs/zh/access.md:53]: Processing Foundation不用翻译！另外这里fellowship可以翻译成研究奖金

**Review by sproutleaf — CHANGES_REQUESTED** [review]
谢谢！

File: contributor_docs/zh/access.md (hunk 3 above)
Comment (Harrycheng233) [reply contributor_docs/zh/access.md]: 我把它翻译成了“无障碍日”，因为如果直接翻译成“可及日”或“可及性日”听起来有点别扭？（虽然没有和别的翻译统一，但如果有更倾向的word choice的话我及时修改！）

File: contributor_docs/zh/access.md (hunk 2 above)
Comment (Harrycheng233) [reply contributor_docs/zh/access.md]: 好滴！

File: contributor_docs/zh/access.md [hunk 5]
```diff
@@ -1,40 +1,58 @@
...
-这是关于p5.js中访问性和包容性的持续对话的一部分。我们的意图是将这些作为p5.js构建的核心价值观，这一点在我们的[社区声明](../CODE_OF_CONDUCT.md)中有所说明，该声明是在[2015年贡献者会议](https://p5js.org/community/contributors-conference-2015.html)上编写的。
+# 我们关注的重点是可及性
 
-**请将此视为一个起点。** 我们希望邀请更多关于可及性意味着什么以及如何优先考虑可及性的讨论。
+在[ 2019 年贡献者大会](https://p5js.org/community/contributors-conference-2019.html)上，我们承诺只向 p5.js 添加提高可及性（包括包容性和无障碍性）的功能。我们不会接受那些不支持这些努力的功能请求。我们致力于承认、废除和预防障碍的工作。这意味着要考虑在交叉[^1]体验中可能影响可及性和参与度的多样性方向。这包括了性别、种族、族裔、性别倾向、语言、地理位置等因素的考虑。我们会优先考虑边缘群体的需求，而不是仅维护在 p5.js 社区中特权群体的长期舒适体验。我们正共同探索和研究可及性的含义，并积极学习该如何实践和传授可及性相关的知识。我们选择通过广泛、交叉和联合的框架来思考可及性。这一承诺是我们在[社区声明](https://p5js.org/community/)中概述的 p5.js 核心价值观的一部分。
 
 ## 可及性的种类
 
-增加可及性并不是关注在扩大p5.js社区的人数。它关注的是使p5.js对那些被排除在p5.js社区之外（无论是有意还是无意）以及类似工具和社区之外的人们更加可用和易于接触。
+增加可及性并不专注于扩大 p5.js 社区的人数。它是一种持续的承诺，旨在使 p5.js 对因结构性压迫而被排除在 p5.js 社区外的人们更加可用和易于接触。这一承诺扩展到 p5.js 提供的工具和平台。它还包括 p5.js 领导层的组成、决策和行动。我们反对技术文化中对速度、增长和竞争的追求。我们将有意识、缓慢、适应性和负责任作为集体关爱的行动优先考虑。
```
Comment (unicar9) [inline contributor_docs/zh/access.md:9]: 我们将有意识、缓慢、适应性和负责任作为集体关爱的行动优先考虑。-> 我们将致力于推行以下互相关照的集体行为：深思熟虑、放慢步调、互相照应和积极负责。（稍微意译让中文更通顺自然。）

File: contributor_docs/zh/access.md [hunk 6]
```diff
@@ -1,40 +1,58 @@
...
-在[2019年贡献者大会](https://p5js.org/community/contributors-conference-2019.html)上，我们承诺只向p5.js添加增加可及性（包括包容性和/或无障碍性）的功能。这意味着要考虑多样性的方向（如性别、社会、经济、种族、族裔、语言、残疾等），这些方向可能影响到可及性/参与度，并采取行动来承认、消除和预防障碍。我们将优先考虑历史上受边缘化群体的需求，而不是为了使相对特权的群体在使用p5.js时更加舒适。
-
-我们不会接受不支持增加可及性努力的功能请求。您将在我们的问题和拉取请求模板中看到这些标准的体现。
+<!-- 我们对可及性的承诺对贡献者和库的用户意味着什么。 -->
 
-这是关于p5.js中访问性和包容性的持续对话的一部分。我们的意图是将这些作为p5.js构建的核心价值观，这一点在我们的[社区声明](../CODE_OF_CONDUCT.md)中有所说明，该声明是在[2015年贡献者会议](https://p5js.org/community/contributors-conference-2015.html)上编写的。
+# 我们关注的重点是可及性
 
-**请将此视为一个起点。** 我们希望邀请更多关于可及性意味着什么以及如何优先考虑可及性的讨论。
+在[ 2019 年贡献者大会](https://p5js.org/community/contributors-conference-2019.html)上，我们承诺只向 p5.js 添加提高可及性（包括包容性和无障碍性）的功能。我们不会接受那些不支持这些努力的功能请求。我们致力于承认、废除和预防障碍的工作。这意味着要考虑在交叉[^1]体验中可能影响可及性和参与度的多样性方向。这包括了性别、种族、族裔、性别倾向、语言、地理位置等因素的考虑。我们会优先考虑边缘群体的需求，而不是仅维护在 p5.js 社区中特权群体的长期舒适体验。我们正共同探索和研究可及性的含义，并积极学习该如何实践和传授可及性相关的知识。我们选择通过广泛、交叉和联合的框架来思考可及性。这一承诺是我们在[社区声明](https://p5js.org/community/)中概述的 p5.js 核心价值观的一部分。
```
Comment (unicar9) [inline contributor_docs/zh/access.md:5]: 我们致力于承认、废除和预防障碍的工作。-> 我们致力于识别、消除和预防与可及性相关的障碍。

**Review by sproutleaf — APPROVED** [review]

**Review by unicar9 — APPROVED** [review]

**Review by sproutleaf — APPROVED** [review]
Approve. Tagging @Qianqianye @limzykenneth for merging. Thank you!

**Comment by limzykenneth** [comment]
@Harrycheng233 @unicar9 @sproutleaf There's a merge conflict here, please do have a look. Thanks.

**Comment by sproutleaf** [comment]
> @Harrycheng233 @unicar9 @sproutleaf There's a merge conflict here, please do have a look. Thanks.

@Harrycheng233 is unable to resolve the conflicts (error message below). Would it be better to delete the existing file and push a new file in 2 separate PRs?
complex

**Comment by limzykenneth** [comment]
@sproutleaf I think I know what's up here, I changed the language code used in this repo from `zh` to `zh-Hans` which likely caused the conflict here with git not able to find the file being changed in this PR. I can resolve this and merge, no worries.

**Pull request merged**